classes/password.php

```diff
@@ -130,6 +94,54 @@ function password_hash($password, $algo, array $options = array()) {
         return $ret;
     }
 
+
+    /**
+     * Generates Entropy using the safest available method, falling back to less preferred methods depending on support
+     *
+     * @param int $bytes
+     *
+     * @return string Returns raw bytes
+     */
+    function generate_entropy($bytes){
```

Comment:
Made more sense to create a separate function for the entropy. Allows this to be called and use some rather decent code anywhere else in the app.